webpack.config.js

```diff
@@ -0,0 +1,142 @@
+var webpack = require('webpack'),
+    path = require('path'),
+    autoprefixer = require('autoprefixer'),
+    Clean = require("clean-webpack-plugin"),
+    AssetsPlugin = require('assets-webpack-plugin'),
+    ExtractTextPlugin = require('extract-text-webpack-plugin'),
+    OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin'),
+    cssnano = require('cssnano');
+
+var SAGE_ENV = process.env.SAGE_ENV || 'development',
```

Comment:
I used SAGE_ENV because the -d argument of webpack can't be passed to its node.js api (in dev.js).

I made a first [commit](https://github.com/ptrckvzn/sage/commit/de5a782f7375661ef302b7f4cb79f3b19e1cf561) to my branch to begin addressing this.
It removes SAGE_ENV and the config settings [output.pathinfo, debug and uglifyjs] that are now set by -d/-p.

I found not equivalent to webpack -d in its node.js API used in dev.js. When webpack -d is run, [it set output.pathinfo and debug to true](http://webpack.github.io/docs/cli.html#development-shortcut-d), but those should be set manually when webpack is called through the node.js API (in dev.js). I thought it could be cleaner to simply put those options into the developement section in webpack.config.js and not bother with webpack -d.